metricbeat/scripts/config_collector.py

```diff
@@ -28,22 +24,24 @@
 # https://www.elastic.co/guide/en/beats/metricbeat/index.html
 
 #==========================  Modules configuration ============================
-metricbeat.modules:
-
 """
 
 
-def collect(beat_path, full=False):
+def collect(beat_name, beat_path, full=False):
 
     base_dir = beat_path + "/module"
     path = os.path.abspath(base_dir)
 
     # yml file
 
     if full:
-        config_yml = header_full
+        config_yml = "########################## " + beat_name + " Configuration ###########################" + header_full
```

Comment:
I tested the branch locally and these changes are needed:
```
     if full:
-        config_yml = "########################## " + beat_name + " Configuration ###########################" + header_full
+        config_yml = "########################## " + beat_name.title() + " Configuration ###########################\n" + header_full
     else:
-        config_yml = "###################### " + beat_name + " Configuration Example #######################" + header_short
+        config_yml = "###################### " + beat_name.title() + " Configuration Example #######################\n" + header_short
```

See also the newline I added at the end. With these changes, also the build should go green.